app/controllers/turbo/native/navigation.rb

```diff
@@ -1,20 +1,23 @@
 # Turbo is built to work with native navigation principles and present those alongside what's required for the web. When you
-# have Turbo Native clients running (see the Turbo iOS and Turbo Android projects for details), you can respond to native
-# requests with three dedicated responses: <tt>recede</tt>, <tt>resume</tt>, <tt>refresh</tt>.
+# have Hotwire Native clients running (see the Hotwire Native iOS and Hotwire Native Android projects for details),
+# you can respond to native requests with three dedicated responses: <tt>recede</tt>, <tt>resume</tt>, <tt>refresh</tt>.
 #
-# turbo-android handles these actions automatically. You are required to implement the handling on your own for turbo-ios.
+# Hotwire Native Android and Hotwire Native iOS handle these actions automatically.
 module Turbo::Native::Navigation
   extend ActiveSupport::Concern
 
   included do
-    helper_method :turbo_native_app?
+    helper_method :hotwire_native_app?, :turbo_native_app?
```

Comment:
It breaks the following patch for Devise:

https://github.com/hotwired/turbo-rails/issues/507#issuecomment-1816664359

Quick fix for anyone stumbling across:

```
class TurboFailureApp < Devise::FailureApp
  # Compatibility for Turbo::Native::Navigation
  class << self
    def helper_method(*names)
    end
  end

  include Turbo::Native::Navigation

  # Turbo Native requests that require authentication should return 401s to trigger the login modal
  def http_auth?
    turbo_native_app? || super
  end
end
```

Reply:
Thanks @ZilvinasKucinskas :)